anilist.py

```diff
@@ -194,43 +195,29 @@ def match_to_plex(anilist_series: List[AnilistSeries], plex_series_watched: List
                     )
 
                     if not match:
+                        # Create first match dict for this anilist id
                         anilist_matches.append({
                             "anilist_id": matched_id,
-                            "watched_episodes": plex_season.watched_episodes,
-                            "total_episodes": plex_season.last_episode,
+                            "watched_episodes": mapped_start + plex_season.watched_episodes,
                             "mapped_seasons": [plex_season.season_number],
-                            "duplicate_mappings": 0,
                         })
                         continue
-                    # For multiple seasons with the same anilist id check how they should be added together
-                    elif (index != 0 and plex_season.first_episode > plex_seasons[index - 1].last_episode):
-                        match["watched_episodes"] = plex_season.watched_episodes
+                    # For multiple seasons with the same id
+                    # If the start of this season has been mapped use that
+                    if mapped_start:
```

Comment:
```suggestion
                    if mapped_start > 0:
```

Integers with value 0 are inherently `false`, but I'd rather write it like this for readability.

Also with your current branch, the `if` is always entered since custom_mappings.py sets `start` to `1` if `start` was not specified. `1` is `true`, that's why the else is never called.

Reply:
It is a bit confusing to do this mapped_start when the default is 1.
I think I got it right now.